internal/clients/team3/clientconfig.go

```diff
@@ -0,0 +1,20 @@
+package team3
+
+func getislandParams() islandParams {
+	return islandParams{
+		giftingThreshold:            45,  // real number
+		equity:                      0.1, // 0-1
+		complianceLevel:             0.1, // 0-1
+		resourcesSkew:               1.3, // >1
+		saveCriticalIsland:          true,
+		escapeCritcaIsland:          true,
+		selfishness:                 0.3, //0-1
+		minimumRequest:              30,  // real number
+		disasterPredictionWeighting: 0.4, // 0-1
+		recidivism:                  0.3, // 0-1
```

Comment:
0.3 makes sense but the range actually goes in the negative. You can day -3 -> 5 (that's a bit arbitrary but it is a good range based on my desmos experimentation)

Reply:
so it takes real number with no bound ?

Reply:
yup